packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -16,19 +16,20 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Table-headCell {
-  color: var(--spectrum-table-header-text-color);
+  color: var(--spectrum-alias-text-color);
```

Comment:
The text color in the XD file is #000000/rgb(0,0,0), but this var resolves to rgb(75,75,75) on light. Do we need to adjust the value of the var? Or is it just that we need to update our spectrum vars?

Reply:
update our spectrum vars
for any reviewers who don't already know, you can get the spectrum plugin for XD, then when you click on a component, it'll tell you the spectrum variable names. you can swap between version 5 and 6

Reply:
Is this the same issue I'm seeing where the sort direction indicator looks darker in the XD file and v5 colors says it should be #6d6d6, closest Spectrum being gray-700, but we're using #8e8e8e/rgb(142,142,142). When I look at the spectrum token of that color we're using gray-600, which isn't gray-700.